code/frameworks/svelte-vite/package.json

```diff
@@ -52,7 +52,7 @@
     "@storybook/svelte": "7.1.0-alpha.27",
     "@sveltejs/vite-plugin-svelte": "^2.0.0",
     "magic-string": "^0.27.0",
-    "svelte": "^3.0.0",
+    "svelte": "^3.0.0 || ^4.0.0 || ^4.0.0-next.0",
```

Comment:
Package managers will install the latest version. Defining these version ranges doesn't make therefore much sense. We have two options here

1. Either we define version 4 and the latest version will always be installed
2. We move this to the peer dependency section (BREAKING CHANGE?!) and adjust the CLI to automatically install svelte for svelte-vite framework.

Reply:
oh yeah this should be peer, it's probably a regular dependency by mistake, option 2 would be the right call.

@shilman would that be a breaking change?

Reply:
Let's make it a peerDependency and a devDependency, similar to vite/react
It makes sense.

svelte users of storybook will have svelte as a dependency... surely?!

Reply:
Yeah I came to the same conclusion @ndelangen, I can't think of a scenario where users wouldn't have Svelte installed already. Maybe in a specific monorepo setup where it's in another package, but I don't know.

I'd consider this more of a bug fix than a breaking change.

Reply:
@ndelangen, `@storybook/sveltekit` depends on `@storybook/svelte-vite` (it's a thin wrapper around it), so making `svelte` a peerDep in `@storybook/svelte-vite` would mean we should do the same in `@storybook/sveltekit` right? Currently there's no dependency on `svelte` at all in `@storybook/sveltekit`.

Reply:
yes, correct